lifecycle-operator/controllers/options/keptnconfig_controller.go

```diff
@@ -99,6 +153,43 @@ func (r *KeptnConfigReconciler) reconcileOtelCollectorUrl(config *optionsv1alpha
 	return ctrl.Result{}, nil
 }
 
+func (r *KeptnConfigReconciler) reconcileRestApi(ctx context.Context, config *optionsv1alpha1.KeptnConfig) (ctrl.Result, error) {
+	if config.Spec.RestApiEnabled {
```

Comment:
I'd change the public name. Since we call it `gateway`, I'd stick to that. So please, adapt also https://github.com/keptn/lifecycle-toolkit/blob/main/lifecycle-operator/apis/options/v1alpha1/keptnconfig_types.go#L57